Pokedex_ios/Assets.xcassets/Pokemons/Bulbasaur.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Is this image needed?

Reply:
i deleted this image too you can check it again